precompiles/erc20/approve.go

```diff
@@ -163,7 +164,7 @@ func (p Precompile) DecreaseAllowance(
 
 	// TODO: owner should be the owner of the contract
```

Comment:
what about these open TODOs that have been in the file since it was merged originally?